ocrd/ocrd/processor/base.py

```diff
@@ -104,6 +105,8 @@ def run_cli(
         args += ['--output-file-grp', output_file_grp]
     if parameter:
         args += ['--parameter', parameter]
+    if overwrite:
+        args += ['--overwrite', overwrite]
```

Comment:
IIUC this is not an option which itself takes an argument, but only a flag:
```suggestion
        args += ['--overwrite']
```